notebooks/official/migration/UJ1 Vertex SDK AutoML Image Classification.ipynb

```diff
@@ -0,0 +1,1413 @@
+{
+  "cells": [
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "copyright"
+      },
+      "outputs": [],
+      "source": [
+        "# Copyright 2021 Google LLC\n",
+        "#\n",
+        "# Licensed under the Apache License, Version 2.0 (the \"License\");\n",
+        "# you may not use this file except in compliance with the License.\n",
+        "# You may obtain a copy of the License at\n",
+        "#\n",
+        "#     https://www.apache.org/licenses/LICENSE-2.0\n",
+        "#\n",
+        "# Unless required by applicable law or agreed to in writing, software\n",
+        "# distributed under the License is distributed on an \"AS IS\" BASIS,\n",
+        "# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.\n",
+        "# See the License for the specific language governing permissions and\n",
+        "# limitations under the License."
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "title:migration,new"
+      },
+      "source": [
+        "# Vertex AI: Vertex AI Migration: AutoML Image Classification\n",
+        "\n",
+        "<table align=\"left\">\n",
+        "  <td>\n",
+        "    <a href=\"https://colab.research.google.com/github/GoogleCloudPlatform/ai-platform-samples/blob/master/vertex-ai-samples/notebooks/official/migration/UJ1 Vertex SDK AutoML Image Classification.ipynb\">\n",
```

Comment:
The link will not work if the notebook file name has space in it, you will have to add "%20" wherever there is space. I would suggest change the file name to "UJ1_Vertex_SDK_AutoML_Image_Classification.ipynb" In that way the name of the notebook can be directly used in the colab/github link.

Reply:
OK, was following the naming convention used by F33 for the original notebooks.